apps/judicial-system/web/src/routes/Court/RestrictionRequest/Overview/OverviewForm.tsx

```diff
@@ -121,128 +127,88 @@ const OverviewForm: React.FC<Props> = (props) => {
-                  </Text>
-                </Box>
-                <Text whiteSpace="breakSpaces">{workingCase.caseFacts}</Text>
-              </Box>
-            )}
-            {workingCase.legalArguments && (
-              <Box marginBottom={2}>
-                <Box marginBottom={2}>
-                  <Text variant="eyebrow" color="blue400">
-                    Lagarök
-                  </Text>
-                </Box>
-                <Text whiteSpace="breakSpaces">
-                  {workingCase.legalArguments}
-                </Text>
-              </Box>
-            )}
-          </div>
-        )}
-        {(workingCase.comments || workingCase.caseFilesComments) && (
-          <div className={styles.infoSection}>
-            <Box marginBottom={2}>
-              <Text variant="h3" as="h2">
-                Athugasemdir
-              </Text>
-            </Box>
-            {workingCase.comments && (
-              <Box marginBottom={workingCase.caseFilesComments ? 3 : 0}>
-                <Box marginBottom={1}>
-                  <Text variant="h4" as="h3" color="blue400">
-                    Athugasemdir vegna málsmeðferðar
-                  </Text>
-                </Box>
-                <Text whiteSpace="breakSpaces">{workingCase.comments}</Text>
-              </Box>
+                .split('\n')
+                .map((requestedCustodyRestriction, index) => {
+                  return (
+                    <div key={index}>
+                      <Text>{requestedCustodyRestriction}</Text>
+                    </div>
+                  )
+                })}
+            </AccordionItem>
+            {(workingCase.caseFacts || workingCase.legalArguments) && (
+              <AccordionItem
+                labelVariant="h3"
+                id="id_4"
+                label="Greinargerð um málsatvik og lagarök"
+              >
+                {workingCase.caseFacts && (
+                  <AccordionListItem title="Málsatvik">
+                    <Text whiteSpace="breakSpaces">
+                      {workingCase.caseFacts}
+                    </Text>
+                  </AccordionListItem>
+                )}
+                {workingCase.legalArguments && (
+                  <AccordionListItem title="Lagarök">
+                    <Text whiteSpace="breakSpaces">
+                      {workingCase.legalArguments}
+                    </Text>
+                  </AccordionListItem>
+                )}
+              </AccordionItem>
             )}
-            {workingCase.caseFilesComments && (
-              <>
-                <Box marginBottom={1}>
-                  <Text variant="h4" as="h3" color="blue400">
-                    Athugasemdir vegna rannsóknargagna
-                  </Text>
-                </Box>
-                <Text whiteSpace="breakSpaces">
-                  {workingCase.caseFilesComments}
-                </Text>
-              </>
+            {(Boolean(workingCase.comments) ||
```

Comment:
Is `Boolean` needed?

Reply:
No, probably not. Checking for string values has always been a pain point for us and this was probably done in an effort to solve it. I removed all Boolean string checks.